src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -42,6 +42,9 @@ public class DataAccessRequest {
   @JsonProperty
   public DataAccessRequestData data;
 
+  @JsonProperty
+  public String darCode;
```

Comment:
Field to hold the dar_code value.

Reply:
I assumed that a DAR would have a dar_code field directly. Since it doesn't, is `darCode` needed for email sending? Or could `referenceId` be used instead?

Reply:
The value in darCode is the promoted way for users to think about these things.  The juice is definitely worth the squeeze on this one.